content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,173 @@
+* Network: A fast network connection is preferable
+
+If you are running `rippled` in an Amazon EC2 instance, Ripple recommends selecting a **Type** of **_m3.large_**. If you choose to run a validating `rippled` server, you may need more resources.
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the software prerequisites you need to build and run `rippled`.
+
+1. Update the list of packages that are available for `apt-get` to install or upgrade.
+```
+sudo apt-get update
+```
+2. Upgrade currently installed packages.
+```
+sudo apt-get -y upgrade
+```
+3. Install Git.
+```
+sudo apt-get -y install git
+```
+4. Install SCons.
+```
+sudo apt-get -y install scons
+```
+5. Install Ctags.
+```
+sudo apt-get -y install ctags
+```
+6. Install `pkg-config`.
+```
+sudo apt-get -y install pkg-config
+```
+7. Install Protocol Buffers.
+```
+sudo apt-get -y install protobuf-compiler
+sudo apt-get -y install libprotobuf-dev
+```
+8. Install Secure Socket Layer (SSL) toolkit development files.
+```
+sudo apt-get -y install libssl-dev
+```
+9. Install `python-software-properties`.
+```
+sudo apt-get install -y python-software-properties
+```
+10. Install Boost.
+```
+sudo add-apt-repository -y ppa:boost-latest/ppa
+sudo apt-get -y update
+sudo apt-get install -y libboost-all-dev
+```
+11. Get `rippled` source code.
+```
+git clone https://github.com/ripple/rippled.git
+cd rippled
+git checkout master
+```
+12. Build `rippled` binary executable from source code. This may take about 30 minutes.
+
+    SCons saves the built executable in `rippled/build`.
+
+```
+scons
+```
+
+
+## 2. Configure `rippled`
+
+1. Configure `rippled.cfg`.
+
+    `rippled.cfg` is the configuration file for `rippled`.
+
+    You can find the `rippled-example.cfg` example file in `rippled/doc`. See the file for a description of all configuration options. You should be able to connect to the XRP Ledger using the default configuration defined in the example file.
+
+    Put your configured `rippled.cfg` file in `etc/opt/ripple/`.
+
+2. Configure `validators.txt`.
+
+    When you start your `rippled` server for the first time, it uses an initial list of validators defined in `validators.txt` to retrieve validation information and to form its UNL (unique node list). The information in `validators.txt` also enables `rippled` clients to indirectly locate IPs they can use to contact the Ripple network.
+
+    You can find the `validators-example.txt` example file in `rippled/doc`. See the file for a description of all validator options. You should be able to connect to the XRP Ledger using the default validators defined in the example file.
+
+    Put your configured `validators.txt` file in `etc/opt/ripple/`.
+
+
+## 3. Run `rippled`
+
+To run your stock `rippled` server from the executable you built, using the configurations you defined:
+```
+cd build
+sudo ./rippled
+```
+
+
+### What to Expect
+
```

Comment:
I added this to help prepare the user for what they see in the terminal once they are running rippled. I would have found this useful when I first started rippled. But, for it to be truly useful, I think we need to add a few explanations of what you are seeing.